### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by damccorm**
Generated from M162

### CODE REVIEW

File: api/DashboardApi.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export interface IDashboardApi extends basem.ClientApiBase {
     createDashboard(dashboard: DashboardInterfaces.Dashboard, teamContext: TfsCoreInterfaces.TeamContext): Promise<DashboardInterfaces.Dashboard>;
     deleteDashboard(teamContext: TfsCoreInterfaces.TeamContext, dashboardId: string): Promise<void>;
     getDashboard(teamContext: TfsCoreInterfaces.TeamContext, dashboardId: string): Promise<DashboardInterfaces.Dashboard>;
-    getDashboards(teamContext: TfsCoreInterfaces.TeamContext): Promise<DashboardInterfaces.DashboardGroup>;
```
Comment (stephenmichaelf) [inline api/DashboardApi.ts]: Breaking change so good to bump major.

File: api/interfaces/BuildInterfaces.ts [hunk 2]
```diff
@@ -1123,14 +1123,18 @@ export enum BuildReason {
...
     BuildCompletion = 512,
+    /**
+     * The build was started when resources in pipeline triggered it
+     */
+    ResourceTrigger = 1024,
     /**
      * The build was triggered for retention policy purposes.
      */
-    Triggered = 943,
+    Triggered = 1967,
```
Comment (stephenmichaelf) [inline api/interfaces/BuildInterfaces.ts]: Scary/surprising to me that enum value changed.

File: api/DashboardApi.ts (hunk 1 above)
Comment (damccorm) [reply api/DashboardApi.ts]: Good catch - did update major, but was questioning that decision, but you're definitely right

**Comment by damccorm** [comment]
> Make sure to run all the samples and make sure they are running successfully.

Yep, looks like they are

**Pull request merged**